Example/Sources/Info.plist

```diff
@@ -16,6 +16,19 @@
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Viewer</string>
+			<key>CFBundleURLName</key>
+			<string></string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>com.virtusize.Example.virtusize</string>
+			</array>
+		</dict>
+	</array>
```

Comment:
One of the steps to enable SNS authentication
![Screen Shot 2021-11-10 at 10 56 52](https://user-images.githubusercontent.com/7802052/141035467-d41ff7e8-1e5a-4137-bf49-b5603ab0f530.png)